gulpfile.js

```diff
@@ -269,14 +271,14 @@ gulp.task(
 gulp.task(
   'watch',
   gulp.series(function() {
-    gulp.watch(paths.src, ['modules']);
+    gulp.watch(paths.src, gulp.parallel('modules'));
   }),
 );
 
 gulp.task(
   'dev',
   gulp.series(function() {
-    gulp.watch(paths.src, ['modules', 'dist']);
+    gulp.watch(paths.src, gulp.parallel('modules', 'dist'));
   }),
```

Comment:
Isn't this redundant? The `dist` task has a dependency on `modules`, and it should be run in series, not in parallel anyway.

Reply:
done